runtime/darwinia/src/pallets/staking.rs

```diff
@@ -12,24 +12,24 @@ use darwinia_staking::{Config, EraIndex};
 #[cfg(feature = "dev")]
 frame_support::parameter_types! {
 	pub const BondingDurationInEra: BlockNumber = 2;
-	pub const BondingDurationInBlockNumber: BlockNumber = 2 * SESSIONS_PER_ERA * BLOCKS_PER_SESSION;
+	pub const BondingDurationInBlockNumber: BlockNumber = 2 * CRAB_SESSIONS_PER_ERA * CRAB_BLOCKS_PER_SESSION;
```

Comment:
I guess it should be `DARWINIA_SESSIONS_PER_ERA` and `DARWINIA_BLOCKS_PER_SESSION`.